SETUP_ARCH.md

```diff
@@ -0,0 +1,23 @@
+# Snowdrift setup instructions for Arch Linux
+
+Arch has the benefit of being very well-documented, so this document is
+very short!
+
+1.  [Install Haskell][1]. While you are at it, make sure you have the
+    `base-devel` group installed.
+2.  [Install PostgreSQL][2]. You don't need to go through setting up a
+    user or anything like that. Snowdrift will do that for you.
+3.  Run these commands:
+
+        git clone https://git.gnu.io/snowdrift/snowdrift.git
+        cd snowdrift
+        ln -s cabal.config.7.10 cabal.config
+        cabal sandbox init
+        cabal install -j -fdev --enable-tests
```

Comment:
I never had it myself either and was surprised when he reported it… I guess we should get people to test this and hopefully simplify the other instructions accordingly.